### ISSUE TITLE
Hide when no active  just like facebook message and notification.

### ISSUE THREAD

**Issue by sunchanras** [malte-wessel/react-custom-scrollbars#16]
Hi @malte-wessel,
Can I hide scroll bar when no active just like Facebook message and notification do.
If there is no auto hide scroll bar feature like FB, can you give me some hint on how to add such feature.

Best Regards,
Sun Chanras

**Comment by malte-wessel** [comment]
Right now you could do something like this:

```JavaScript
const AutoHideScrollbars = createClass({

  getInitialState() {
    return {
      hide: true
    };
  },

  handleMouseEnter() {
    this.setState({ hide: false });
  },

  handleMouseLeave() {
    this.setState({ hide: true });
  },

  renderScrollbar({ style, ...props }) {
    const { hide } = this.state;
    const finalStyle = {
      ...style,
      transition: 'opacity 200ms',
      opacity: hide ? 0 : 1
    };
    return <div style={finalStyle} {...props}/>
  },

  render() {
    return (
      <Scrollbars
        onMouseEnter={this.handleMouseEnter}
        onMouseLeave={this.handleMouseLeave}
        renderScrollbarHorizontal={this.renderScrollbar}
        renderScrollbarVertical={this.renderScrollbar}
        {...this.props}/>
    );
  }
});
```
I'm planning to add auto hide support in v3.0.0. Follow #17

**Comment by malte-wessel** [comment]
Auto hide mode is available in v3.0.0. Check out the [documentation on auto hide](https://github.com/malte-wessel/react-custom-scrollbars/blob/master/docs/usage.md#auto-hide) as well as the [upgrade guide](https://github.com/malte-wessel/react-custom-scrollbars/blob/master/docs/upgrade-guide-v2-v3.md)
